lib/SearchAndSortPane/SearchAndSortPane.test.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { waitFor } from '@testing-library/dom';
+import {
+  render, screen, fireEvent,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  buildMutator, buildResources,
+} from '../../test/helpers';
+
+import '../../test/jest/__mock__';
+import '../../test/jest/__new_mock__';
+
+import { SettingsLabel } from '../Settings';
+import { SearchAndSortPane } from './SearchAndSortPane';
+
+describe('SearchAndSortPane', () => {
+  const resultCountMessage = 'profiles pane';
+  const titleMessageId = 'title message';
+  const searchAndSortPaneCommonProps = {
+    hasSearchForm: true,
+    visibleColumns: ['id', 'name', 'hrId'],
+    columnMapping: {
+      id: 'id',
+      name: 'Name',
+      hrId: 'hrId',
+    },
+    initialResultCount: 1,
+    resultCountIncrement: 2,
+    resultsFormatter: {
+      id: record => record.id,
+      name: record => record.name,
+    },
+    parentResources: buildResources({
+      resourceName: 'profiles',
+      records: [
+        {
+          id: 1,
+          name: 'Test name',
+        },
+        {
+          id: 2,
+          name: 'Test name 2',
+        },
+      ],
+    }),
+    parentMutator: buildMutator(),
+    resourceName: 'profiles',
+    defaultSort: 'name',
+    resultCountMessageId: resultCountMessage,
+    label: (
+      <SettingsLabel
+        messageId={titleMessageId}
+        iconKey="customIcon"
+      />
+    ),
+  };
+
+  describe('rendering component with loaded status and 2 items', () => {
+    beforeEach(async () => {
+      window.history.pushState({}, document.title, '/');
+    });
+
+    afterEach(async () => {
+      window.history.pushState({}, document.title, '/');
+    });
+
+    beforeEach(async () => {
+      await render(
```

Comment:
As we discussed early, move render in the It-block. Because use it in before each not best practice.